apigw/src/internal/mobile-device-session.ts

```diff
@@ -101,12 +101,14 @@ export const pinLoginRequestHandler = (redisClient: RedisClient) =>
     const employeeId = assertStringProp(req.body, 'employeeId')
     const response = await employeePinLogin(req)
 
-    const token = uuid()
-    await redisClient.set(toMobileEmployeeIdKey(token), employeeId, {
-      EX: pinSessionTimeoutSeconds
-    })
+    if (response.status === 'SUCCESS') {
+      const token = uuid()
+      await redisClient.set(toMobileEmployeeIdKey(token), employeeId, {
+        EX: pinSessionTimeoutSeconds
+      })
+      req.session.employeeIdToken = token
+    }
```

Comment:
any need for logging in else-branch?

Reply:
bäkkärissä on jo täysi audit-lokitus statuksesta riippumatta, eli tässä ei ole tarvetta duplikoida sitä